S2VX.Game/Story/Note.cs

```diff
@@ -29,19 +52,55 @@ public class Note : RelativeBox {
                 // Return early to save some calculations
                 return;
             }
+            var startTime = EndTime - notes.ShowTime;
+            var startFadeIn = startTime - notes.FadeInTime;
 
-            Rotation = camera.Rotation;
-            Size = camera.Scale;
-            Position = S2VXUtils.Rotate(Coordinates - camera.Position, Rotation) * Size.X;
+            SquareNote.Rotation = camera.Rotation;
+            SquareNote.Size = camera.Scale;
+            SquareNote.Position = S2VXUtils.Rotate(Coordinates - camera.Position, SquareNote.Rotation) * SquareNote.Size.X;
```

Comment:
position, rotation, and scale variables are defined below. Let's do a bit of refactoring to move these variables up so we don't need to recall camera.Position, camera.Rotation, camera.Scale, etc. like this.

Reply:
Ok moved